beep/collate.py

```diff
@@ -229,6 +229,10 @@ def process_files_json():
     # chdir into beep root
     pwd = os.getcwd()
     os.chdir(os.environ.get("BEEP_ROOT", "/"))
```

Comment:
If we're going to make directories for the user, we may want to change the default BEEP_ROOT to something we're more likely to have permissions in, like the repo itself.  We also may, as the issue notes, want to think about a more descriptive name, e. g. BEEP_PROCESSING_DIR, or something along those lines.

Reply:
Agreed. I will change the name to something a little more descriptive. I think we can set it so that certain environments default to the beep installation location and the `stage` and `prod` environments default to the root